test/e2e/common/apparmor.go

```diff
@@ -84,7 +84,7 @@ done`, testCmd)
 		Spec: api.PodSpec{
 			Containers: []api.Container{{
 				Name:    "test",
-				Image:   "gcr.io/google_containers/busybox:1.24",
+				Image:   busyboxImage,
```

Comment:
What do we feel about referencing some custom `busyboxImage` variable vs _always_ calling `imageutils`?
To me it seems like always calling the SSOT makes sense, regardless of if we there prefix things with `-GOARCH` or not.

I don't think the `busyboxImage` shorthands makes things much shorter actually, as there are duplicates everywhere (a new BusyboxImage var has to be created in every new package, instead of always referencing the canonical place)